app/src/organisms/OnDeviceDisplay/RobotDashboard/RecentRunProtocolCard.tsx

```diff
@@ -3,6 +3,7 @@ import { css } from 'styled-components'
 import { useTranslation } from 'react-i18next'
 import { useHistory } from 'react-router-dom'
 import { formatDistance } from 'date-fns'
+import { last } from 'lodash'
```

Comment:
nit: importing this way is better so you don't import the whole lodash library

```suggestion
import last from 'lodash/last'
```